plugins/modules/clustering/znode.py

```diff
@@ -121,7 +126,8 @@ def main():
             op=dict(choices=['get', 'wait', 'list']),
             state=dict(choices=['present', 'absent']),
             timeout=dict(default=300, type='int'),
-            recursive=dict(default=False, type='bool')
+            recursive=dict(default=False, type='bool'),
+            usessl=dict(default=False, type='bool')
```

Comment:
Will make the diff nicer the next time an option is added:
```suggestion
            usessl=dict(default=False, type='bool'),
```